packages/react-components/src/localization/locales/en-US/strings.json

```diff
@@ -49,7 +49,7 @@
     "tooltipOffContent": "Turn on camera",
     "tooltipVideoLoadingContent": "Video is loading",
     "cameraMenuTitle": "Camera",
-    "cameraMenuTooltip": "Choose Camera",
+    "cameraMenuTooltip": "Choose camera",
```

Comment:
Is this something that also needs to be done in the composite strings as well?

Reply:
Yes. But I did not find any tooltip string in the composite. So I think we are good :)